pkgs/development/python-modules/shtab/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "shtab";
+  version = "1.4.2";
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi {
+    pname = "shtab";
+    inherit version;
+    sha256 = "sha256-oR8pa/FJ3ywcx4GUHkhRkSmK0bdF3BHQHLImxVWv98s=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "use_scm_version=True" "version='${version}'"
+    sed -i -e 's/setuptools>=[0-9\.]*;//' setup.cfg
+    sed -i -e 's/setuptools_scm\[toml\]>=[0-9\.]*//' setup.cfg
```

Comment:
Do we really need those overrides? They are both >= so this should work with our never versions.

Reply:
It's not about the versions, it's about removing setuptools_scm as this requires having network connectivity during the build which fails in the sandbox.
But I don't need to remove setuptools, so I will keep that one in. I only need to remove the setuptools_scm string [here](https://github.com/iterative/shtab/blob/eb60f65335bc66ba36f5cf654f5f4fb4b9b8f2f8/setup.cfg#L69).

Reply:
Thats strange. setuptools_scm is used a lot in pythonPackages and I never saw that it requires internet connectivity. What does it try to download? Do we need to add that afterwards?

Reply:
Reading the output carefully I realized that setuptools_scm itself was missing! Adding it to the buildInputs fixed the problem. Thanks! I have updated the PR.